.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,5 +12,4 @@ Resolves #
 - [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
 - [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
 - [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
-- [ ] `make vendor` does not cause changes.
```

Comment:
Doesn't the user still need check if there are changes?

Reply:
I don't believe there should be any changes unless someone modifies `Gopkg.toml` manually.

Reply:
If the code have reference to third party dependency which is located on GOPATH instead of vendor dir, the user can still build the program. And contributor may not check whether there is any need to update vendor dir. Since this is a task that can be done by CI build pipeline and I think this is a good idea to automate this step. I would like to open a pr for this.